synapse/models/risk.py

```diff
@@ -150,13 +161,30 @@ def getModelDefs(self):
                     'doc': 'A taxonomy of threat types.'}),
 
                 ('risk:leak', ('guid', {}), {
+                    'display': {
+                        'columns': (
+                            {'type': 'prop', 'opts': {'name': 'disclosed'}},
+                            {'type': 'prop', 'opts': {'name': 'name'}},
+                            {'type': 'prop', 'opts': {'name': 'owner::name'}},
+                            {'type': 'prop', 'opts': {'name': 'reporter:name'}},
```

Comment:
I think we should probably prefer `reporter:name` here vs `reporter::name` ( which is consistent with other similar use cases )